service/test/lib/integration/teamOpsTest.js

```diff
@@ -5,188 +5,259 @@ const _ = require('lodash')
 const expect = require('code').expect
 const Lab = require('lab')
 const lab = exports.lab = Lab.script()
+const crypto = require('crypto')
 
 const teamOps = require('../../../lib/ops/teamOps')
 const policyOps = require('../../../lib/ops/policyOps')
 const userOps = require('../../../lib/ops/userOps')
 
-let testTeamId
-const teamData = {
-  name: 'Team 4',
-  description: 'This is a test team',
-  parentId: null,
-  organizationId: 'WONKA'
-}
-const updateData = {
-  name: 'Team 5',
-  description: 'description',
-  organizationId: 'WONKA'
-}
-const updateTeamMembersData = {
-  users: ['MikeId', 'CharlieId'],
-  organizationId: 'WONKA'
+function randomId () {
+  return crypto.randomBytes(2).toString('hex')
 }
 
-
 lab.experiment('TeamOps', () => {
 
-  lab.before((done) => {
-    teamOps.createTeam(teamData, function (err, result) {
```

Comment:
I could see a createTeam in `beforeEach` but not in `before`.
Typically these tests update the team. If all tests share the same team it's going to be hard to remember what a team should be before and after each test so that all tests work well together.